ivcfmerge/batch.py

```diff
@@ -29,8 +30,16 @@ def ivcfmerge_batch(input_paths, output_path, batch_size=None, temp_dir=None):
 
     _ivcfmerge(batch, output_path)
 
+    if temp_dir:
+        clean_up_temp_dir(temp_dir)
+
 
 def _ivcfmerge(input_paths, output_path):
     with ExitStack() as stack, open(output_path, 'w+') as outfile:
         infiles = map(lambda p: stack.enter_context(open(p)), input_paths)
         ivcfmerge(infiles, outfile)
+
+
+def clean_up_temp_dir(temp_dir):
+    for path in Path(temp_dir).glob('*'):
```

Comment:
I don't think you can remove everything under the temp_dir. What if the temp_dir happens to contain other files?

Reply:
@Zhicheng-Liu Fixed! Thanks!

Reply:
@Zhicheng-Liu also added a test to ensure this condition: https://github.com/iqbal-lab-org/ivcfmerge/pull/8/files#diff-71f839c3e1c7e69cdcc25d7da2c1860eR81